mdoc/src/main/scala-3/mdoc/internal/io/ConsoleColors.scala

```diff
@@ -0,0 +1,9 @@
+package mdoc.internal.io
+
+case class ConsoleColors(
+    green: String => String = identity,
```

Comment:
The idea is not to use it now, since we will be able to upgrade much more rapidly and not wait for the fansi release. We might come back to it once we have a stable binary version.

Reply:
:+1: